gcp-config-values.yml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+opentelemetry-collector:
```

Comment:
for my own knowledge, why do we need both this collector config, and the collector config in otelcollector/?

Reply:
Can you add comments to clarify?  It seems like there are significant differences between them.  Do can you make sure to document why those differences exist?

Reply:
added comments